sdks/go/pkg/beam/runners/dataflow/dataflowlib/job.go

```diff
@@ -82,6 +84,21 @@ func Translate(p *pb.Pipeline, opts *JobOptions, workerURL, modelURL string) (*d
 		return nil, fmt.Errorf("Dataflow supports one container image only: %v", images)
 	}
 
+	packages := []*df.Package{{
+		Name:     "worker",
+		Location: workerURL,
+	}}
+	experiments := append(opts.Experiments, "beam_fn_api")
```

Comment:
When the job is created, it seems we just passes a path pointing to the jar (rather than the actual jar file) to dataflow service side, right?

Reply:
Right. It is a pointer to a GCS object of the form:

  gs://foo/bar/baz